tests/dragonfly/generic_test.py

```diff
@@ -24,7 +24,7 @@ def export_dfly_password() -> str:
 
 @pytest.mark.asyncio
 async def test_password(df_local_factory, export_dfly_password):
-    dfly = df_local_factory.create()
+    dfly = df_local_factory.create(proactor_threads=1)
```

Comment:
Why it's important?

Reply:
because we test only control commands, and If I'm running it from a 16 thread cpu it'll require 4gb of ram

Reply:
lgtm